lib/y2_installbase.pm

```diff
@@ -550,7 +550,12 @@ sub save_remote_upload_y2logs {
 sub post_fail_hook {
     my $self = shift;
 
-    if (check_var("REMOTE_CONTROLLER", "ssh") || check_var("REMOTE_CONTROLLER", "vnc")) {
+    if (check_screen 'error_during_installation') {
+        # In case of error during installation collect y2logs right away
+        get_to_console;
+        $self->save_upload_y2logs;
```

Comment:
but this flow only save y2logs, we should get the rest of the stuff as well, right? does it work moving `get_to_console()` before SUPER?